tools/codegen-diff-revisions.py

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/env python3
```

Comment:
rip bash 🥳

Reply:
Bash was doomed the moment I needed a function with arguments...